src/utils/local.ts

```diff
@@ -0,0 +1,8 @@
+import UkLocal from "../components/UkLocal.vue";
+
+export function getLocalComponent(country:string) {
+  if(country === "UK"){
+    return UkLocal;
+  }
+  return null;
+}
```

Comment:
Ideally, something like a `Option<T>` or `Maybe<T>` would be used instead of `null`, but YMMV in TypeScript.

Reply:
This is checked to not display if `null`